src/sql/workbench/contrib/notebook/browser/cellViews/output.component.ts

```diff
@@ -54,7 +54,6 @@ export class OutputComponent extends CellView implements OnInit, AfterViewInit {
 	ngOnInit() {
 		this._register(this._themeService.onDidColorThemeChange(event => this.updateTheme(event)));
 		this.loadComponent();
-		this.layout();
 		this._initialized = true;
 		this._register(Event.debounce(this.cellModel.notebookModel.layoutChanged, (l, e) => e, 50, /*leading=*/false)
```

Comment:
Is there still any noticeable perf issues when this event is fired? What triggers this to happen?

Reply:
Layout is triggered when the ADS window size is changed. There is a noticeable lag while resizing the ADS window when a large notebook is open vs when a small notebook is open. That is likely caused by the layout of all the notebook contents being recalculated.